### PR TITLE
KAFKA-15157: Print startup time for RemoteIndexCache

### PR DESCRIPTION
**Pull request by DL1231**
https://issues.apache.org/jira/projects/KAFKA/issues/KAFKA-15157
Print startup time for RemoteIndexCache.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/log/remote/RemoteIndexCache.scala [hunk 1]
```diff
@@ -178,6 +178,7 @@ class RemoteIndexCache(maxSize: Int = 1024, remoteStorageManager: RemoteStorageM
     .build[Uuid, Entry]()
 
   private def init(): Unit = {
+    val start = time.hiResClockMs()
```
Comment (divijvaidya) [inline core/src/main/scala/kafka/log/remote/RemoteIndexCache.scala]: you can use `Time.SYSTEM.hiResClockMs()`

**Comment by divijvaidya** [comment]
@satishd FYI, this is also a candidate to be incorporated into https://github.com/apache/kafka/pull/13275

File: core/src/main/scala/kafka/log/remote/RemoteIndexCache.scala (hunk 1 above)
Comment (DL1231) [reply core/src/main/scala/kafka/log/remote/RemoteIndexCache.scala]: Done.

**Review by divijvaidya — APPROVED** [review]

**Review by satishd — APPROVED** [review]
Thanks @DL1231 for the PR. LGTM

**Pull request merged**